mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -33,6 +33,7 @@ CLASS aun net/minecraft/entity/player/PlayerEntity
 	FIELD g spawnPosition Leu;
 	METHOD a sendVillagerRecipes (ILbfc;IIZ)V
 		ARG 1 syncId
+		ARG 5 canLevel
```

Comment:
Isn't this like showing level in screen which is true for villagers but false for wandering traders?

Reply:
Yep